### ISSUE TITLE
Add setting for default google domain

### ISSUE THREAD

**Issue by sammcj** [containers-everywhere/contain-google#44]
Right now the google container always goes to the American Google site www.google.com.

This is a bit annoying if you want google containerised but still want to have results more related to your country.

It would be great if in preferences you could set the default Google domain, e.g. www.google.com.au (for Australia).

**Comment by Perflyst** [comment]
If I open `http://www.google.com.au/` I dont get redirect to google.com. Does this still happen for you? Can you try in a new browser profile, please?

**Comment by sammcj** [comment]
That’s correct, but if you search from the address bar you get given google.com, not your browsers locate based google (.com.au for Australia)

--
Sam McLeod

**Comment by sammcj** [comment]
Sorry I should have added I tried with stable, beta and nightly - nightly which had a new profile, it’s also the same on all 3 of my machines if that helps.

Let me know if there’s anything else I can do to help.

--
Sam McLeod

**Comment by Perflyst** [comment]
What if you explicit use a open search form for .com.au like https://addons.mozilla.org/en-US/firefox/addon/google-com-au? Note, this is not an addon, this is just an additional search option.

**Comment by sammcj** [comment]
Just AFK at the moment - I’ll test tomorrow morning.

Thanks again for your assistance!

--
Sam McLeod

**Comment by sammcj** [comment]
@Perflyst, after installing https://addons.mozilla.org/en-US/firefox/addon/google-com-au setting the contain-google does search with google.com.au.

So I'm wondering if when you install the contain-google addon it clears or ignores a setting that https://addons.mozilla.org/en-US/firefox/addon/google-com-au adds back in?
